JavaFXUserInterface/src/main/resources/AssetInfo.fxml

```diff
@@ -106,7 +158,10 @@
         </AnchorPane>
     </children>
     <stylesheets>
-        <URL value="@css/revamp/assetInfo.css" />
-        <URL value="@css/revamp/style.css" />
+        <URL value="@css/assetInfo.css"/>
```

Comment:
IntelliJ cannot resolve the filename 'assetInfo.css'.
There's a typo, just change it to 'assetinfo.css to match the file name :)

Reply:
![image](https://user-images.githubusercontent.com/11137499/114293113-7fb4e980-9a61-11eb-96be-ef1a7ec94695.png)
I don't have this issue, and it's spelt the exact same way on my end

Reply:
<img width="784" alt="image" src="https://user-images.githubusercontent.com/17152773/114293550-0a4b1800-9a65-11eb-96a9-c00420286a9a.png">
weird....

Reply:
ok resolved now!

Reply:
Fixed